<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockManageWidget</class>
 <widget class="QWidget" name="StockManageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pbnModify">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Open_pushButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/images/open.png);</string>
         </property>
         <property name="text">
          <string>下架</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Save_pushButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/images/save.png);</string>
         </property>
         <property name="text">
          <string>上架</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="check_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/images/ok.png);</string>
         </property>
         <property name="text">
          <string>退货或补货</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Delete_Row_pushButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>条形码标签</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>筛选项目：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbxSift">
             <item>
              <property name="text">
               <string>商品条码</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>商品名称</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>商品规格</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>商品颜色</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>筛选内容：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="letContent">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbnFind">
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbnShowAll">
             <property name="text">
              <string>全部显示</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tableView"/>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labInfo">
              <property name="text">
               <string>共有 0 条数据   共 0 页   当前第 0 页</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnFirst">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>第一页(&amp;F)</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/first.png</normaloff>:/images/first.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnPre">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>上一页(&amp;M)</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/pre.png</normaloff>:/images/pre.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnNext">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>下一页(&amp;N)</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnLast">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>末一页(&amp;L)</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/last.png</normaloff>:/images/last.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>100</width>
                <height>23</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
